tests/python/test_empty.py

```diff
@@ -8,3 +8,12 @@ def func():
     pass
 
   func()
+
+@ti.all_archs
+def test_empty_args():
+
+  @ti.kernel
+  def func(x: ti.i32, arr: ti.ext_arr()):
+    pass
+
+  foo(42, np.arange(10, dtype=np.float32))
```

Comment:
Did you mean to call func in the test?

Reply:
Yeah, I think so. `foo` maybe directly copied from the example code of #531.

Reply:
Nice catch! Done